sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/google/GoogleHttpClient.java

```diff
@@ -54,20 +54,19 @@
     private static final String HTTP_MULTIPART_PART1_DISPOSITION_FORMAT = "form-data; name=\"%s\"; filename=\"blob\"";
     private static final String HTTP_MULTIPART_PART1_DISPOSITION_ENTITY_VALUE = "entity";
     private static final String HTTP_MULTIPART_PART2_DISPOSITION_FORMAT = "form-data; name=\"content\"; filename=\"%s\"";
+    private static final int HTTP_REQUEST_RETRY_COUNT = 1;
 
     private final Logger logger = LogManager.getLogger(GoogleHttpClient.class.getName());
     private final HttpRequestFactory requestFactory;
     private String lwssoValue = "";
     private final String urlDomain;
+    private Authentication lastUsedAuthentication;
 
-    /**
-     * Creates an HTTP client instance using the url and authentication.
-     *
-     * @param urlDomain The source URL of the Octane server
-     * @param clientTypeHeader The special client type that can be sent to the API
-     */
-    public GoogleHttpClient(final String urlDomain, final String clientTypeHeader) {
+    public GoogleHttpClient(final String urlDomain) {
         this.urlDomain = urlDomain;
+
+        //String clientTypeHeader = authentication.getClientHeader();
+
         HttpTransport HTTP_TRANSPORT = new NetHttpTransport();
         requestFactory = HTTP_TRANSPORT
                 .createRequestFactory(request -> {
```

Comment:
Remember that we wanted to perhaps extract this lambda out into a separate method to make it easier to understand